### DISCUSSION TITLE
[WARNING GRT-0350] Missing route to pin iregister_read/io_bypass_1_bits_data[0].

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD#4163] [General]
This is for the top level of MegaBoom that I'm playing around with.

Is this warning telling me that it can't find a route for the buffer tree for the pin in the warning?

It looks like this buffer tree is going from the iregister_read macro to the ALUExeUnit, which traverses Rob(reorder buffer), rename stage and the memory issue unit. The mpl2 macro placement is unfortunate here...

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/e3c6b965-bf83-403c-9dd5-70413735a8ca)

![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/2798822/c2a76f06-f286-4b5a-8bc7-2fc0219bc9f8)

**Comment by maliberty** [comment]
@eder-matheus please respond

**Comment by eder-matheus** [comment]
@oharboe @maliberty This warning is related to the estimate_parasitics using global route parasitics. It usually happens when a routing topology is not correctly translated to the parasitics estimation structures. I will look into the test case and fix the cause of this bug.

**Reply by oharboe** [reply]
I see. That answers my question.
